docker/Dockerfile.faux

```diff
@@ -10,9 +10,9 @@ RUN $AG update && $AG install openjdk-8-jre
 
 RUN $AG update && $AG install openjdk-8-jdk git
 
-ENV BACHASH=d97c917a5ad24e6c9b45f49a5b01efffda9e820e
+ENV BACHASH=a560a116135c576a5cb787a3be7a9763f4a2b2e9
```

Comment:
we need to make sure this hash is updated in the other places it's used (search for BACHASH).  Ideally we should make the ENV variable passed in, but that's another chunk of work. Kinda depends on if we think this would ever change.

Reply:
Looks like docker/Dockerfile.test_bacnet is the other usage.